src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -81,24 +81,20 @@ export class UrlGeneratorService {
     switch (profile.type) {
-      case ProfileType.SPACE:
-      case ProfileType.CHALLENGE:
-      case ProfileType.OPPORTUNITY:
+      case ProfileType.SPACE_ABOUT:
         return await this.getSpaceUrlPathByAboutProfileID(profile.id);
-      case ProfileType.USER: {
+      case ProfileType.USER:
         const userEntityInfo = await this.getNameableEntityInfoForProfileOrFail(
           'user',
           profile.id
         );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)